### PR TITLE
Initial Re-org

### PR DESCRIPTION
**Pull request by Anyhowclick**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,27 +0,0 @@
-{
```
Comment (ilanDoron) [inline .prettierrc:1]: lets keep this file for js pretify

we want to pretify all js files.
such that prettier runs by default.

File: package.json [hunk 2]
```diff
@@ -10,10 +10,12 @@
     "test": "./tst.sh",
     "t": "./tst.sh",
     "compile": "./cmp.sh",
-    "compileSol6": "./cmpSol6.sh",
```
Comment (ilanDoron) [inline package.json:13]: note:
Above version should be 2.0.0

not sure why I can't comment.
Seems need to reopen PR with new master.


also not sure why main is truffle.js
should probably change.

File: package.json [hunk 3]
```diff
@@ -30,29 +32,30 @@
     "compare-versions": "3.5.1",
     "dotenv": "8.2.0",
     "ethers": "5.0.4",
-    "ganache-cli": "6.8.2",
```
Comment (ilanDoron) [inline package.json:33]: please try to remove unused packages.

compare-versions.
ethers?
Etc

File: solcOptimiserSettings.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
 module.exports = {
     enabled: true,
-    runs: 430
+    runs: 30000
```
Comment (ilanDoron) [inline solcOptimiserSettings.js:3]: can't go higher??

**Review by ilanDoron — COMMENTED** [review]
see a few comments.

File: solcOptimiserSettings.js (hunk 4 above)
Comment (Anyhowclick) [reply solcOptimiserSettings.js:3]: It's possible, but I think orderbook or eth2dai reserve will exceed max bytecode size, causing tests to fail

**Pull request closed without merging**